recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   md5: dd01e07a77123c128ff79ba57b97c1d7
 
 build:
-  number: 0
+  number: 1
```

Comment:
Not sure that it needs a build number bump. Added one anyways since we have changed the recipe. Though could be convinced to remove it. 😄